src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -200,7 +201,9 @@ public Collection<Range<Token>> getPrimaryRangesWithinDC(String keyspace)
     /* the probability for tracing any particular request, 0 disables tracing and 1 enables for all */
     private double traceProbability = 0.0;
 
-    private static enum Mode { STARTING, NORMAL, JOINING, LEAVING, DECOMMISSIONED, MOVING, DRAINING, DRAINED }
+    private static enum Mode
```

Comment:
this file got accidentally reformatted :). Or was it intentional?

Reply:
I don't think it was intentional. I need to take a look at whether there's an ant task to reformat files "the cassandra way", otherways this is getting tiresome :-)